app/templates/teachers/convert-to-teacher-account-view.jade

```diff
@@ -28,40 +28,36 @@ block content
       .row.m-y-2
         .col-md-offset-2.col-md-4.col-sm-6
           .form-group
-            label.control-label(data-i18n="general.username")
+            span.control-label(data-i18n="general.username")
```

Comment:
I needed the validation errors to be on the same line, right-justified.  Is there a simple way to do that with labels?